src/chains/ethereum/options/src/logging-options.ts

```diff
@@ -74,6 +73,16 @@ export type LoggingConfig = {
       type: boolean;
       hasDefault: true;
     };
+
+    /**
+     * If you set this option, Ganache will write logs to a file located at the
+     * specified path.
+     * Note: If you provide a `URL` it must use the `path://` protocol.
```

Comment:
The tone/style of this doc comment is different than the rest of the options docs comments. Maybe something like:

```
The file append logs to.

Can be a filename, url, or descriptor.

If specifying a url, the path must begin with `path://`.

By default no log file is created.
```

Reply:
Does this accept a string that starts with `path://`, or does it have to be a `URL`?

Reply:
It does not - I've clarified that in the description. We _could_ implement this, but I feel like it's not going to be very widely used.